OneSignalExample/Packages/com.onesignal.unity.core/Editor/Core/Manifest.cs

```diff
@@ -0,0 +1,166 @@
+        /// </summary>
+        /// <param name="name">Name of the dependency.</param>
+        /// <returns>Dependency with given name.</returns>
+        public Dependency GetDependency(string name)
+        {
+            return m_Dependencies[name];
+        }
+
+        /// <summary>
+        /// Returns scope registry by a provided url.
+        /// </summary>
+        /// <param name="url">Scope registry url.</param>
+        /// <returns>Scope registry with the given url.</returns>
+        public ScopeRegistry GetScopeRegistry(string url)
+        {
+            return m_ScopeRegistries[url];
+        }
+
+        /// <summary>
+        /// Adds scope registry.
+        /// </summary>
+        /// <param name="registry">An entry to add.</param>
+        public void AddScopeRegistry(ScopeRegistry registry)
+        {
+            if (!IsRegistryExists(registry.Url))
+            {
+                m_ScopeRegistries.Add(registry.Url, registry);
+            }
+        }
+
+        /// <summary>
+        /// Adds dependency.
+        /// </summary>
+        /// <param name="name">Dependency name.</param>
+        /// <param name="version">Dependency version.</param>
+        public void AddDependency(string name, string version)
+        {
+            if (!IsDependencyExists(name))
+            {
+                var dependency = new Dependency(name, version);
+                m_Dependencies.Add(dependency.Name, dependency);
+            }
+        }
+
+        /// <summary>
+        /// Writes changes back to the manifest file.
+        /// </summary>
+        public void ApplyChanges()
+        {
+            var registries = new List<object>();
+            foreach (var registry in m_ScopeRegistries.Values)
+            {
+                registries.Add(registry.ToDictionary());
+            }
+            m_RawContent[k_ScopedRegistriesKey] = registries;
+
+            // Remove 'scopedRegistries' key from raw content if we have zero scope registries.
+            // Because we don't need an empty 'scopedRegistries' key in the manifest
+            if (registries.Count == 0)
+                m_RawContent.Remove(k_ScopedRegistriesKey);
+
+            Dictionary<string,object> dependencies = new Dictionary<string, object>();
+            foreach (var dependency in m_Dependencies.Values)
+            {
+                dependencies.Add(dependency.Name, dependency.Version);
+            }
+            m_RawContent[k_DependenciesKey] = dependencies;
+
+            string manifestText = Json.Serialize(m_RawContent, true);
+            File.WriteAllText(Path,manifestText);
+        }
+
+        /// <summary>
+        /// Searches for ScopeRegistry with the provided Url.
+        /// </summary>
+        /// <param name="url">ScopeRegistry url to search for.</param>
+        /// <returns>`true` if scoped registry found, `false` otherwise.</returns>
+        public bool IsRegistryExists(string url)
```

Comment:
Same fixes from the `OneSignal-Unity-Bootstrap` repo need to be applied here.
Should be `IsRegistryPresent`